src/components/card/EmissionsCard.tsx

```diff
@@ -0,0 +1,78 @@
+import type { FC } from 'react'
+import React from 'react'
+import clsx from 'clsx'
+
+import CarIcon from '../icon/CarIcon'
+import PlugIcon from '../icon/PlugIcon'
+import TreeIcon from '../icon/TreeIcon'
+import TrashIcon from '../icon/TrashIcon'
+
+export interface CardProps {
+  title: string
+  metric: string
+  description: string
+  color: string
+  type: string
+}
+
+interface IconProps {
+  type: string
+}
+
+const Icon: FC<IconProps> = ({ type }) => {
+  let icon
+
+  switch (type) {
+    case 'gasoline':
+      icon = <CarIcon />
+      break
+    case 'trashbag':
+      icon = <TrashIcon />
+      break
+    case 'home':
+      icon = <PlugIcon />
+      break
+    case 'tree':
+      icon = <TreeIcon />
+      break
+  }
+
+  return <div className="px-3"> {icon}</div>
+}
+
+const EmissionsCard: FC<CardProps> = ({
+  title,
+  metric,
+  description,
+  color,
+  type,
+}) => {
+  return (
+    <div className="py-4">
+      <h2 className="subheader-1 py-6">{title}:</h2>
+      <div
+        className={clsx('rounded-md border-2', {
+          'border-pumpkin': color === 'pumpkin',
+          'border-brightTeal': color === 'brightTeal',
+        })}
+      >
+        <div className="flex items-center space-x-4 py-4">
+          <div
+            className={clsx('ml-4 flex items-center justify-center', {
+              'bg-clementine/20': color === 'pumpkin',
+              'bg-lightBlue': color === 'brightTeal',
+            })}
+          >
+            <text className="h-6 min-w-[120px] text-center text-xl font-black">
+              {metric}
+            </text>
+          </div>
+          <div className="text-lg leading-5">{description}</div>
```

Comment:
```suggestion
        <div className="flex items-center justify-between py-4">
          <div className="flex items-center">
            <div
              className={clsx('ml-4 flex items-center justify-center', {
                'bg-clementine/20': color === 'pumpkin',
                'bg-lightBlue': color === 'brightTeal',
              })}
            >
              <text className="h-6 min-w-[120px] text-center text-xl font-black">
                {metric}
              </text>
            </div>
            <div className="ml-4 text-lg leading-5">{description}</div>
          </div>
```

This will help keep all the icons aligned as so
![image](https://github.com/toriis-portal/toriis/assets/73120774/172fb7ef-7b0e-4962-9f79-35cc37489017)

Instead of the unaligned current version
![image](https://github.com/toriis-portal/toriis/assets/73120774/fc9d2761-688c-40fe-b525-9f36161cace8)